wallet/udb/addressmanager.go

```diff
@@ -29,7 +29,7 @@ const (
 	// not exceed the hardened child range of extended keys and it provides
 	// a reserved account at the top of the range for supporting imported
 	// addresses.
-	MaxAccountNum = hdkeychain.HardenedKeyStart - 2 // 2^31 - 2
+	MaxAccountNum = hdkeychain.HardenedKeyStart - 3 // 2^31 - 2
```

Comment:
Instead of reducing the available number of BIP0044 accounts we can derive, we should be using account numbers higher than HardenedKeyStart